coverage/functions.csv

```diff
@@ -1,11 +1,11 @@
-AuthenticationService.ts,2
-ActivesPartsService.ts,5
-ActivesUsersService.ts,3
-Minimax.ts,1
-NodeUnheritance.ts,1
-online-game-wrapper.component.ts,2
-PieceThreat.ts,1
-PylosState.ts,1
-QuartoRules.ts,1
-server-page.component.ts,1
-SixMinimax.ts,3
+ActivesPartsService.ts,5
+ActivesUsersService.ts,3
+AuthenticationService.ts,2
+Minimax.ts,1
+NodeUnheritance.ts,1
+online-game-wrapper.component.ts,1
+PylosState.ts,1
+PieceThreat.ts,1
+QuartoRules.ts,1

```

Comment:
super louche ce changement d'ordre alphabétique windows/linux :D Genre aucun des deux n'est bon

Reply:
J'avoue, ils ont l'air de ne se soucier que de la première lettre ces baudets.
P-e à ajouter au script ?

Reply:
ah bah tiens oui, si tu enlève le [0] ligne 13 de coverage.py ça devrait être bon du coup !

Reply:
noté! Sinon là j'ai bien les csv mais ils semblent supprimés sur ma branche, et de mon côté, tout se comporte comme si ils étaient gitignorés. Bizarrerie messire duc.

Reply:
1. Check que tu as bien des lignes comme ceci dans ton .gitignore:
```
! coverage/branches.csv
```
2. Essaye avec un git add manuel (git add coverage/branches.csv)
3. Sinon y'a moyen de retravailler le .gitignore pour exclure tout sauf ces CSV (ce qui est fait actuellement avec le !, mais visiblement chez toi ça n'marche pas c'machin là claudy!)

Reply:
bah, ça marchait il me semble, il a pété un boulon et s'est rebellé !